aws/sdk/build.gradle.kts

```diff
@@ -263,6 +273,21 @@ tasks.register<Exec>("fixManifests") {
     dependsOn("relocateExamples")
 }
 
+tasks.register<Exec>("inflateReadme") {
+    description = "Run the publisher tool's `inflate-readme` sub-command to create the final AWS Rust SDK README file"
+
+    inputs.dir(publisherToolPath)
+    outputs.dir(outputDir)
+
+    workingDir(publisherToolPath)
+    commandLine(
+        "cargo", "run", "--","inflate-readme",
+        "--sdk-version", getSdkVersion(),
+        "--msrv", getRustMSRV(),
+        "--output", outputDir.resolve("README.md").absolutePath
+    )
+}
+
```

Comment:
A possibly better term than inflate is hydrate:
- https://en.wikipedia.org/wiki/Hydration_(web_development)
- https://stackoverflow.com/questions/6991135/what-does-it-mean-to-hydrate-an-object

It's what I used to call this sort of thing in web dev world (that or just "template" although I don't quite when the noun and verb for something are the same word.)

Reply:
Probably the word I was reaching for but couldn't quite remember 😄